python/lsst/ts/mthexapod/config_schema.py

```diff
@@ -1,5 +1,35 @@
+# This file is part of ts_mthexapod.
+#
+# Developed for the Rubin Observatory Telescope and Site System.
```

Comment:
I think we agreed on `Vera Rubin Observatory` but I'll leave it up to you to decide if you want to update this and the other files.

Reply:
I think I'll leave it for now. It's close enough.